yew-macro/src/html_tree/html_prop.rs

```diff
@@ -31,7 +37,11 @@ impl Parse for HtmlProp {
         let value = input.parse::<Expr>()?;
         // backwards compat
         let _ = input.parse::<Token![,]>();
-        Ok(HtmlProp { label, value })
+        Ok(HtmlProp {
```

Comment:
I know you didn't change this but [Clippy might complain about not using `Self`](self-explanatory) pretty soon.
```suggestion
        Ok(Self {
```